src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -197,6 +197,30 @@
     <value>Remap shortcuts</value>
     <comment>Keyboard Manager remap shortcuts header</comment>
   </data>
+  <data name="RemapKeysList.AutomationProperties.Name" xml:space="preserve">
+    <value>Current Key Remappings</value>
+  </data>
+  <data name="RemapShortcutsList.AutomationProperties.Name" xml:space="preserve">
+    <value>Current Shortcut Remappings</value>
+  </data>
+  <data name="KeyboardManager_RemappedKeysListItem.AutomationProperties.Name" xml:space="preserve">
+    <value>Key Remapping</value>
+  </data>
+  <data name="KeyboardManager_RemappedShortcutsListItem.AutomationProperties.Name" xml:space="preserve">
+    <value>Shortcut Remapping</value>
+  </data>
+  <data name="KeyboardManager_RemappingIcon.AutomationProperties.Name" xml:space="preserve">

```

Comment:
The text for the arrow icons is not getting read out by Windows Narrator

Reply:
Looks like it just skips the FontIcon altogether

Reply:
I tried it with NVDA and it worked fine. Let me see if it's some narrator related issue or if setting some other property fixes it.

Reply:
@arjunbalgovind, I fixed it. Can you pls review it again? Thanks.